pkg/osinfo/container_scanner.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package osinfo
+
+import (
+	"bufio"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+type ContainerScanner struct{}
```

Comment:
For which reason do we introduce a new structure if it contains nothing? (If reasonable, docs would be good to understand its purpose)

Reply:
The main reason is that I want to split these into their own implementation for fakes, but did not get around to do it. I've added a todo item to make it clear (here and in the layer scanner)